webhook-connector/configure-the-webhook.md

```diff
@@ -4,50 +4,68 @@ description: Configurate the webhook in the Tradecloud portal
 
 # Configure the webhook
 
-## Configure your webhook in the Tradecloud1 portal
+## Configure your webhooks in the Tradecloud1 portal
 
-### Select Webhook
+### Select one or more webhooks
 
-As a company admin configure the webhook in your company settings on the [Tradecloud1 platform](http://portal.tradecloud1.com):
+As a company admin configure one or more webhooks in your company settings on the [Tradecloud1 platform](http://portal.tradecloud1.com):
 
-![](../.gitbook/assets/blue-sailboats-webhook.png)
+![Webhook Settings](../.gitbook/assets/webhook-settings.png)
 
-* Select **My team** in the menu below your avatar.
-* Select **Settings** in the page menu.
-* Select **Webhook** in the Integration section.
+* Select **My company** in the menu below your avatar.
+* The **Settings** tab in the Company menu should be selected.
```

Comment:
Why reword it? imo it's easier to scan this text if the bullets have the same sentence structure.

Reply:
If you select **My company** then **Settings** is already selected.

Reply:
I see. fair enough